eventuals/collect.h

```diff
@@ -16,7 +17,23 @@ template <typename Container>
   return Loop<Container>()
       .context(Container())
       .body([](auto& data, auto& stream, auto&& value) {
-        if constexpr (HasEmplaceBack<Container>::value) {
+        if constexpr (
+            std::is_same_v<
+                Container,
+                google::protobuf::RepeatedField<
+                    typename Container::value_type>>) {
+          data.Add(value);
```

Comment:
std::move(value)

Reply:
Add's argument is `const &`, so no need to move

Reply:
TIL!